src/ducks/User.js

```diff
@@ -1,442 +1,82 @@
 import firebase from "firebase";
 import { Actions } from "react-native-router-flux";
-import { Alert } from "react-native";
 import { createActionTypes } from "../utils/actions";
 
 // handle all things related to Users
 const ACTIONS = createActionTypes([
-	"FIRST_NAME_CHANGED",
-	"LAST_NAME_CHANGED",
-	"EMAIL_CHANGED",
-	"MAJOR_CHANGED",
-	"COLLEGE_CHANGED",
-	"POINTS_CHANGED",
-	"PRIVILEGE_CHANGED",
-	"GET_PRIVILEGE",
-	"PICTURE_CHANGED",
-	"PASSWORD_CHANGED",
-	"CONFIRM_PASSWORD_CHANGED",
-	"REGISTRATION_ERROR",
-	"SET_DASH_COLOR",
-	"SET_FLAG",
 	"SHOW_FIREBASE_ERROR",
 	"VERIFIED_USER",
 	"LOGIN_USER",
-	"GET_USER_COMMITTEES",
-	"CHANGE_USER_COMMITTEES",
-	"ENTER_APP",
-	"LOAD_USER",
-	"LOGIN_USER_FAIL",
 	"LOGOUT_USER",
-	"CREATE_USER",
-	"CREATE_USER_SUCCESS",
-	"CREATE_USER_FAIL",
-	"EDIT_USER",
-	"CONTINENT_CHANGED",
-	"NATIONALITY_CHANGED",
-	"GENDER_CHANGED",
-	"BIRTH_DATE_CHANGED",
-	"GO_TO_RESET_PASSWORD",
-	"GO_TO_LOGIN",
-	"GO_TO_PROFILE",
-	"GO_TO_REGISTRATION",
-	"GO_TO_EDIT_PROFILE_FORM",
-	"QUOTE_CHANGED",
-	"PAGE_LOAD",
-	"RESET_PASSWORD"
+	"LOAD_USER_AND_PRIVILEGE"
 ]);
 
 const INITIAL_STATE = {
-	firstName: "",
-	dashColor: "#21252b",
-	flag: "",
-	lastName: "",
-	email: "",
-	college: "",
-	major: "",
-	quote: "",
-	picture: "",
-	points: 0,
-	privilege: {},
-	password: "",
-	confirmPassword: "",
-	continent: "Do not wish to disclose",
-	nationality: "Do not wish to disclose",
-	gender: "Do not wish to disclose",
-	birthday: "0000-00-00",
-	paidMember: false,
-	user: null,
+	activeUser: {
+		firstName: "",
+		color: "#21252b",
```

Comment:
There is some disagreement in the use of color vs dashColor in this ducks and the other screens. Does this affect the UI at all?

Reply:
Not sure what you mean here. Is dashColor used anywhere else?